Automattic/wp-calypso
# Gutenpack: Break up the Jetpack preset's view bundle into separate, per-block ones.

#### Changes proposed in this Pull Request

* make it so that we can build tile-galleries as a separate bundle. 


#### Testing instructions

run the usual command that builds the bundle. 
`npm run sdk gutenberg client/gutenberg/extensions/presets/jetpack -- --output-dir="../jetpack/_inc/blocks"` 

it should produce the following files.
```
editor.js
editor.css
editor.rtl.css
tiled-gallery/view.js
tiled-gallery/view.css
tiled-gallery/view.rtl.css
```
